### PR TITLE
Code refactored to fix codacy issues and reduce code complexity.

### PR DESCRIPTION
**Pull request by DhobaleVishal**
Signed-off-by: Vishal Dhobale <vishal.dhobale@seagate.com>

# Problem Statement
-   Code refactored to fix codacy issues and reduce code complexity.
Ref: https://app.codacy.com/gh/Seagate/corio/files 

# Design
-   For Bug, Describe the fix here.
-   For new test addition post the link for design

# Coding
   Checklist for Author
-   [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-   [x] New/Affected tests are executed
-   [x] Attach test execution logs

# Review 
  Checklist for Author
-   [x] JIRA number/GitHub Issue added to PR
-   [x] PR is self reviewed
-   [x] Jira and state/status is updated and JIRA is updated with PR link
-   [x] Check if the description is clear and explained

# Documentation 
  Checklist for Author
-   [ ] Changes done to ReadMe

-----
[View rendered .github/pull_request_template.md](https://github.com/DhobaleVishal/corio/blob/CORTX-30411_codacy_fixes_from_dev/.github/pull_request_template.md)

### CODE REVIEW

**Review by akshaym99 — APPROVED** [review]

File: src/commons/scheduler.py [hunk 1]
```diff
@@ -95,15 +95,3 @@ def schedule_test_plan(test_plan: str, test_plan_values: dict, common_params: di
         LOGGER.info("%s Loop interrupted", process_name)
         loop.stop()
     LOGGER.info("%s terminating", process_name)
-
-
-def terminate_processes(process_list):
```
Comment (priyankaborawake) [inline src/commons/scheduler.py:100]: Any reason for removing this ?

Comment (DhobaleVishal) [reply src/commons/scheduler.py:100]: kept in corio package.

**Comment by ketanarlulkar** [comment]
@DhobaleVishal, this is too much change in one single PR and it's difficult to review. Please try to avoid such a huge change in single PR.

**Comment by ketanarlulkar** [comment]
Please attach execution logs.

**Comment by DhobaleVishal** [comment]
> @DhobaleVishal, this is too much change in one single PR and it's difficult to review. Please try to avoid such a huge change in single PR.

@ketanarlulkar Yes agree, but this is expected as I have fixed most of the codacy issues.

**Review by kulrk — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Comment by DhobaleVishal** [comment]
> Please attach execution logs.

https://jts.seagate.com/secure/attachment/516265/CORTX-30411_codacy_fixes_from_dev.zip

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**